### DISCUSSION TITLE
s there simple way to continue training a TTS model from an existing checkpoint ?

### DISCUSSION THREAD

**Discussion by besacier** [coqui-ai/TTS#1615] [General Q&A] [answered]
hi
is there simple way to continue training a TTS model from an existing checkpoint ?
best
L

**Comment by thorstenMueller** [comment, marked as answer]
Do you know trainings cli arguments `restore_path` and/or `continue_path` ?

* https://tts.readthedocs.io/en/latest/finetuning.html?highlight=restore_path#steps-to-fine-tune-a-tts-model
* https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html?highlight=continue_path#pure-python-way
